test/test_functional_tensor_cleanup.py

```diff
@@ -0,0 +1,145 @@
+import unittest
+
+import torch
+import torchvision.transforms.functional_tensor as F_t
+
+
+class Tester(unittest.TestCase):
+    def setUp(self):
+        self.device = "cpu"
+
+    def test_get_image_size(self):
+        scripted_fn = torch.jit.script(F_t._get_image_size)
+
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor)
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+        shape = (100, 100)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        self.assertTrue([shape[-1], shape[-2]], scripted_fn(tensor))
+
+    def test_vflip(self):
+        scripted_fn = torch.jit.script(F_t.vflip)
```

Comment:
Not sure if it's necessary to jit-script the methods in this test. Feel free to omit this if you find issues during the refactoring.